README.md

```diff
@@ -1,77 +1,82 @@
+<a href="https://www.tickgit.com/browse?repo=github.com/godotengine/godot"><img src="https://badgen.net/https/api.tickgit.com/badgen/github.com/godotengine/godot" alt="TODOs"></a></p>
 
-**[Godot Engine](https://godotengine.org) is a feature-packed, cross-platform
+<h3 align="center">
+ A 2D and 3D cross-platform game engine
+</h3>
+	
+
+- **[Godot Engine ↗](https://godotengine.org) is a feature-packed, cross-platform
 game engine to create 2D and 3D games from a unified interface.** It provides a
-comprehensive set of [common tools](https://godotengine.org/features), so that users can focus on making games
+comprehensive set of [common tools & features ↗](https://godotengine.org/features), so that users can focus on making games
 without having to reinvent the wheel. Games can be exported with one click to a
 number of platforms, including the major desktop platforms (Linux, macOS,
 Windows), mobile platforms (Android, iOS), as well as Web-based platforms
-and [consoles](https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html).
+and [consoles ↗](https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html).
 
-## Free, open source and community-driven
+## 💸 Free, open source and 🚻 community-driven
 
-Godot is completely free and open source under the very permissive [MIT license](https://godotengine.org/license).
+- Godot is completely **Free** and **Open source** under the free of charge [MIT license ↗](https://godotengine.org/license).
 No strings attached, no royalties, nothing. The users' games are theirs, down
 to the last line of engine code. Godot's development is fully independent and
 community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the [Software Freedom Conservancy](https://sfconservancy.org/)
+expectations. It is also supported by the [Software Freedom Conservancy ↗](https://sfconservancy.org/)
 not-for-profit.
 
-Before being open sourced in [February 2014](https://github.com/godotengine/godot/commit/0b806ee0fc9097fa7bda7ac0109191c9c5e0a1ac),
-Godot had been developed by [Juan Linietsky](https://github.com/reduz) and
-[Ariel Manzur](https://github.com/punto-) (both still maintaining the project) for several
+> Before being open sourced in [February 2014 ↗](https://github.com/godotengine/godot/commit/0b806ee0fc9097fa7bda7ac0109191c9c5e0a1ac),
+Godot had been developed by [Juan Linietsky ↗](https://github.com/reduz) and
+[Ariel Manzur ↗](https://github.com/punto-) (both still maintaining the project) for several
 years as an in-house engine, used to publish several work-for-hire titles.
 
-![Screenshot of a 3D scene in the Godot Engine editor](https://raw.githubusercontent.com/godotengine/godot-design/master/screenshots/editor_tps_demo_1920x1080.jpg)
+![Screenshot of a 3D scene in the Godot Engine editor ↗](https://raw.githubusercontent.com/godotengine/godot-design/master/screenshots/editor_tps_demo_1920x1080.jpg)
```

Comment:
This is image alt text – it doesn't need an external link icon.

Reply:
Oops my bad